docs/02-get-started/08-install-uninstall-upgrade-kyma-module.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Install, uninstall and upgrade Kyma with a module
+---
+
+This guide shows how to quickly install, uninstall or upgrade Kyma with specific modules. To see the list of all available and planned Kyma modules, go to the [Kyma modules](../README.md#kyma-modules).
+
+> **NOTE:** This guide describes installation of a standalone Kyma instance with specific modules, and not how to enable modules in SAP BTP, Kyma runtime (SKR).
+
+## Install Kyma with a module
+
+To enable a module, deploy its module manager and apply the module configuration. The operation installs a standalone Kyma instance with a module of your choice. See the already available Kyma modules with their quick installation steps and links to their GitHub repositories:
```

Comment:
It is getting really interesting here. At the end you will not have a "kyma" instance anymore, you only install modules, that's it. Kyma is just an abstract umbrella here, nothing technical anymore..

Reply:
I got rid of "instance" to make that clear.

Reply:
I think the sentence `The operation installs a standalone Kyma instance with a module of your choice.` can be completely removed as it is not true.

Reply:
already done